### PR TITLE
Sfl 79

### PR DESCRIPTION
**Pull request by aniket-Webonise**

### CODE REVIEW

File: app/src/main/java/com/sjana/fitness/view/feature/home/move/MoveFragment.kt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+package com.sjana.fitness.view.feature.home.move
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+import com.sjana.fitness.R
+
+class MoveFragment : Fragment(){
```
Comment (tejasAlsi-Webonise) [inline app/src/main/java/com/sjana/fitness/view/feature/home/move/MoveFragment.kt:10]: extend with BaseFragment(), refer other fragment structure.

File: app/src/main/res/layout/fragment_move.xml [hunk 2]
```diff
@@ -4,6 +4,9 @@
     android:layout_height="match_parent"
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
-
+    <androidx.recyclerview.widget.RecyclerView
```
Comment (tejasAlsi-Webonise) [inline app/src/main/res/layout/fragment_move.xml:7]: file name should be move_fragment.xml

Comment (aniket-Webonise) [reply app/src/main/res/layout/fragment_move.xml:7]: done

File: app/src/main/java/com/sjana/fitness/view/feature/home/move/MoveFragment.kt (hunk 1 above)
Comment (aniket-Webonise) [reply app/src/main/java/com/sjana/fitness/view/feature/home/move/MoveFragment.kt:9]: it needs a view model when made then will implement this

**Pull request merged**